<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelMainWindow</class>
 <widget class="QMainWindow" name="LevelMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="level_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Study</string>
    </property>
   </widget>
   <widget class="QPushButton" name="level_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Level 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="level_1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Level 1</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/unnamed.jpg);</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>level_3</zorder>
   <zorder>level_2</zorder>
   <zorder>level_1</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
